<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordsListWindow</class>
 <widget class="QDialog" name="WordsListWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Словарь</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#WordsListWindow{
background-color: white;
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>22</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox {
   border: 0px solid gray;
}</string>
   </property>
   <property name="title">
    <string>Сортировать по:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QRadioButton" name="radioButtonRu_Eng">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Русским</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonEng_Ru">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>40</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Английским</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnAddWord">
   <property name="geometry">
    <rect>
     <x>273</x>
     <y>22</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Добавить слово</string>
   </property>
   <property name="styleSheet">
    <string notr="true">#btnAddWord{
border-image: url(:/res/img/add.png);
}

#btnAddWord:hover{
border-image: url(:/res/img/addHover.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="listWords">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>90</y>
     <width>251</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="labelImg">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>90</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelWord">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Удалить слово</string>
   </property>
   <property name="styleSheet">
    <string notr="true">#btnDelWord{
border-image: url(:/res/img/del.png);
}

#btnDelWord:hover{
border-image: url(:/res/img/delHover.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>410</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#btnClose{
color: rgb(58,33,102);
background-color: rgba(160,192,253, 180);
border: 0px solid black;
border-radius: 35px;
}

#btnClose::hover{
	background-color: rgba(160,192,253, 255);
}</string>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWordsCount">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>60</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(66, 200, 98);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelShowing">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>355</y>
     <width>171</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelCorrectly">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>385</y>
     <width>171</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelRating">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>415</y>
     <width>171</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
